server/repository/userRole/update.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseProtocol, DB } from '@server/db'
+import { UserRole, RoleName } from '@meta/user'
+import { Objects } from '@core/utils'
+
+export const update = async (
+  props: {
+    userRole: UserRole<RoleName, any>
+    accept?: boolean
+  },
+  client: BaseProtocol = DB
+): Promise<UserRole<RoleName, any>> => {
+  const {
+    userRole: { id, invitationUuid, acceptedAt },
+    accept = false,
+  } = props
+
+  const _acceptedAt = accept ? `now()` : acceptedAt
+
+  return client.one<UserRole<RoleName, any>>(
+    `
+        update users_role set
+            invitation_uuid = $2,
```

Comment:
why do you set invitation_uuid ? this shouldn't change after user has been invited.
I suspect this method should just be acceptInvitation